src/lib/utils/balancer/helpers/sor/sorManager.ts

```diff
@@ -147,7 +146,7 @@ export class SorManager {
       v2TokenIn.toLowerCase(),
       v2TokenOut.toLowerCase(),
       swapType,
-      BigNumber.from(amountScaled.toString()),
```

Comment:
This is the line that was throwing an exception in prod. `amountScaled` was being passed through as `1.23456e+23` which was coming from the scale function in useSor and then BigNumber didn't know what to do with this value.